### PR TITLE
Changed mixin to an input element (#67)

### PR DESCRIPTION
**Pull request by reisfmb**

### CODE REVIEW

File: src/main/resources/admin/tools/content-type-manager/components/input/component-input.html [hunk 1]
```diff
@@ -81,30 +81,28 @@
...
-            },
-          },
         };
       },
       created() {
         this.map = app.$root._data.map;
+
+        // TODO: Add cache to avoid unecessary requests
+        fetch(
+            `http://localhost:8080/_/service/tool.content.type.manager/test2?schemaName=${this.inputType}`
```
Comment (poi33) [inline src/main/resources/admin/tools/content-type-manager/components/input/component-input.html:92]: localhost 👎

File: src/main/resources/admin/tools/content-type-manager/components/input/fields/component-input-fields-config.html [hunk 2]
```diff
@@ -76,6 +77,24 @@
       },
       created() {
         this.map = app.$root._data.map;
+
+        // TODO: Add cache to avoid unecessary requests
+        fetch(
+            `http://localhost:8080/_/service/tool.content.type.manager/test2?schemaName=${this.inputType}`
```
Comment (poi33) [inline src/main/resources/admin/tools/content-type-manager/components/input/fields/component-input-fields-config.html:83]: ^

**Review by poi33 — CHANGES_REQUESTED** [review]
This is a duplicate of #69.

**Comment by reisfmb** [comment]
> This is a duplicate of #69.

Idk why I added the commit related to the issue #64 :see_no_evil:

**Review by poi33 — APPROVED** [review]

**Pull request merged**